ui/packages/catalog-ui-search/src/main/webapp/component/visualization/table/thead.view.js

```diff
@@ -96,6 +97,14 @@ module.exports = Marionette.ItemView.extend({
         var hiddenColumns = user.get('user').get('preferences').get('columnHide');
         var availableAttributes = this.options.selectionInterface.getActiveSearchResultsAttributes();
 
+        var resultTemplateAttributes = [];
+        if (store.getCurrentQuery()) {
+            var selectedResultTemplate = store.getCurrentQuery().get('selectedResultTemplate');
```

Comment:
❗️ Similar comment about accessing a model property that isn't defined in [Query.js](https://github.com/codice/ddf/blob/master/ui/packages/catalog-ui-search/src/main/webapp/js/model/Query.js#L74-L88)